pkg/auth/db_service.go

```diff
@@ -837,3 +868,372 @@ func (s *DBAuthService) markTokenSingleUse(ctx context.Context, tokenID string,
 	}
 	return canUseToken, nil
 }
+
+var ErrExportedEntNotFound = errors.New("previously exported entitiy not found")
+
+func Migrate(ctx context.Context, d *pgxpool.Pool, writer io.Writer) error {
```

Comment:
Missing installation metadata?

Reply:
Same as above - pending `MetadataManager` support of `KV`

Reply:
Thanks, I just noticed it is standalone and not part of the auth service itself